fplog/fplog.cpp

```diff
@@ -458,7 +456,6 @@ class FPLOG_API Fplog_Impl
 
         struct Filter_Mapping_Entry
```

Comment:
I think now it is better to convert structure to typedef because it seems you don't need structure anymore.
typedef std::map<std::string, Filter_Base*> Filter_Mapping_Entry;